### DISCUSSION TITLE
关于音乐播放器的中文歌词乱码问题

### DISCUSSION THREAD

**Discussion by likepoems** [AlistGo/alist#2953] [Q&A] [answered]
alist 3.6 音乐播放器的歌词中文乱码，英文显示正常。

**Comment by NovaShen0724** [comment, marked as answer]
请在反代中配置字符编码

**Reply by likepoems** [reply]
我在nginx的server里添加charset utf-8;之后网站就打不开了。

**Reply by NovaShen0724** [reply]
请给出报错截图 nginx配置文件 nginx的log(可选)

**Reply by likepoems** [reply]
现在没报错了，只是添加之后不生效。试了添加到server里和location里，直接打开歌词文件也是乱码。

[![pSmAufs.jpg](https://s1.ax1x.com/2023/01/10/pSmAufs.jpg)](https://imgse.com/i/pSmAufs)

**Reply by NovaShen0724** [reply]
看不到你的截图[无力]
要不给个复现链接？您有ipv6吧

**Reply by NovaShen0724** [reply]
看到了看到了！！稍等

**Reply by NovaShen0724** [reply]
这是因为您的歌词文件没有使用utf格式保存，也许ta是ansl，需要转换格式即可，因为这个markdown编辑器就是当成utf8来解码的
参见我的复现链接
[链接](https://nas.novashen.top/test)

**Reply by likepoems** [reply]
感谢，回去我试试。
